dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -44,6 +44,34 @@ rocommunity {{ snmp_rocommunity }}
 rocommunity6 {{ snmp_rocommunity }}
 {% endif %}
 
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY.keys() %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
+rocommunity {{ community }}
+rocommunity6 {{ community }}
+{% else %}
```

Comment:
If community type is not RO, we should not assume it is RW.
Type is mandatory field, so can we assume that it will either be RO or RW.
If there is no Type defined, then we cannot use that community string, is that right?

Reply:
Thanks for the feedback.  I've set this up to have to be either RO or RW or it will not work.

Reply:
Also added the rwcommunity and rwcommunity6 sections.